### ISSUE TITLE
Color-less status bar

### ISSUE THREAD

**Issue by borekb** [KnisterPeter/vscode-github#202]
I would love to be able to configure the status bar to be color-less, to be more in line with the rest of the VSCode UI. I was even thinking about customizing the text, for example, I don't find the word "success" useful there, so maybe something like this?

```
{
  "github.statusBar.color": false,
  "github.statusBar.command": "vscode-github.checkoutPullRequests",
  "github.statusBar.successText": "${icon} ${prNumber}",
  "github.statusBar.errorText": "${icon} ${prNumber} !!!",
}
```

Just an idea.

**Comment by KnisterPeter** [comment]
Nice icea. I'll integrate that in one of the next releases.

**Comment by KnisterPeter** [comment]
The status bar could be colorless and the status texts could be adjusted as well.
There are three variables which could be used:

${icon}
${prNumber}
${status}

This will be in the next release.

**Comment by borekb** [comment]
Thanks for such a quick update! 🎉

**Comment by KnisterPeter** [comment]
Released with 0.21

**Comment by borekb** [comment]
Works nicely, thanks.
